tests/unit/stream_alert_cli/test_outputs.py

```diff
@@ -38,6 +38,8 @@ def test_load_output_config():
         u'aws-firehose',
         u'aws-lambda',
         u'aws-s3',
+        u'aws-sns',
```

Comment:
While you're adding to this - can you remove the `u` unicode prefix from these since it's not required/is confusing.